Bumper release notes

aludvik opened the pull request.

Commit:
Transpose release notes to website markdown

Signed-off-by: Adam Ludvik <ludvik@bitwise.io>

Commit:
Add "Latest Release" section to homepage

Signed-off-by: Adam Ludvik <ludvik@bitwise.io>

annechenette commented on generator/source/index.md:
```diff
@@ -23,6 +23,15 @@ community and documented
 
+# Latest Release
+
+The latest release of Hyperledger Sawtooth is 1.1, codename Bumper.
```
Is it bumper or Bumper?

Also, it should be codenamed (not codename).

Commit:
Add "Releases" page to track stable releases

Signed-off-by: Adam Ludvik <ludvik@bitwise.io>

dcmiddle commented on generator/source/release/bumper/notes.md:
```diff
@@ -0,0 +1,315 @@
+- State pruning is now supported at a configurable block horizon. This helps
+  limit the total storage requirements for global state by removing historic
+  state after the configured horizon. Check out the [state pruning
+  RFC](https://github.com/hyperledger/sawtooth-rfcs/pull/8) for more details:
```
:s/:/.

annechenette reviewed (commented):
The documentation section appears incomplete - it appears that the only doc changes were to add a Rust SDK and a glossary.  I can provide an additional commit with the formatting changes.

Commit:
Make format consistent in RN/Documentation

Change the format of doc change so that each major item is a subheading,
for consistency and clarity.

Signed-off-by: Anne Chenette <chenette@bitwise.io>

Commit:
Remove "release notes" from link + RN page

Simplify the links on the releases page and change the focus of
the release notes page to a description of the release.

The intent of this change is to provide a release-specific page to describe
the release as a whole, without focusing on terms like "release notes".

Signed-off-by: Anne Chenette <chenette@bitwise.io>

Commit:
Edit release notes

Streamline content, correct format, and fix typos.

Signed-off-by: Anne Chenette <chenette@bitwise.io>
